src/status_im/subs/profile.cljs

```diff
@@ -81,12 +81,20 @@
             :override-ring?  override-ring?
             :font-file       font-file}))}))))
 
+;; DEPRECATED
+;; use `:profile/public-key` instead
 (re-frame/reg-sub
  :multiaccount/public-key
  :<- [:profile/profile]
  (fn [{:keys [public-key]}]
    public-key))
 
+(re-frame/reg-sub
+ :profile/public-key
```

Comment:
Should we add a corresponding test for this?

Reply:
done